test/functional/provider/perl_spec.lua

```diff
@@ -21,13 +21,15 @@ do
   end
 end
 
+if skip(funcs.executable('perl') == 0, 'missing "perl" command') then
+  return
+end
+
```

Comment:
Isn't this already covered by the check above?

Reply:
good catch, yes `missing_provider` handles this;

Reply:
```suggestion
```